app.js

```diff
@@ -4,16 +4,32 @@
  */
 import express from 'express';
 import logger from 'morgan';
+import session from 'express-session';
 
-import userInfosHelper from './helpers/userInfo';
-import idHintHelper from './helpers/idHintToken';
-import logoutHelper from './helpers/logout';
-import authorizationHelper from './helpers/authorization';
+import utilsHelper from './helpers/utils';
 import getAccessTokenHelper from './helpers/accessToken';
 
 const app = express();
 const port = process.env.PORT || '3000';
 
+/**
+ * session config
+ * @type {{secret: string, cookie: {}, saveUninitialized: boolean}}
+ */
+const sess = {
+  secret: 'demo secret', // put your own secret
+  cookie: {},
+  saveUninitialized: true,
+};
+
+/**
```

Comment:
are you sure it's needed ? This project intend to be the simpliest as possible. We must keep in mind that PHPers must understand this code